src/Processors/Transforms/buildPushingToViewsChain.cpp

```diff
@@ -788,6 +791,13 @@ void FinalizingViewsTransform::work()
         auto & status = statuses[i];
         ++i;
 
+        if (status.exception && materialized_views_ignore_errors)
```

Comment:
I'd expect to still see the failure in query_views_log, which I think this is skipping. Maybe merging the 2 ifs would be more clear?